psalm-baseline.xml

```diff
@@ -1,22 +1,22 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<files psalm-version="5.17.0@c620f6e80d0abfca532b00bda366062aaedf6e5d">
+<files psalm-version="5.25.0@01a8eb06b9e9cc6cfb6a320bf9fb14331919d505">
   <file src="src/AbstractSerializer.php">
     <RedundantCondition>
-      <code>! $crFound</code>
-      <code>! $crFound</code>
+      <code><![CDATA[! $crFound]]></code>
+      <code><![CDATA[! $crFound]]></code>
```

Comment:
I wonder if this CDATA thrashing in the baseline is due to different xml libraries available when baseline is updated.

Reply:
Maybe - I have libxml 2.9.13 locally for PHP 8.1